public/javascripts/SVLabel/src/SVLabel/mission/Mission.js

```diff
@@ -346,7 +346,9 @@ function Mission(parameters) {
             return getProperty("distance");
         }
     }
-
+    function setLabelCountsAtCompletion(labelCounts){
```

Comment:
I think this was sticking around from the older code. It's not used so I will get rid of it